nbs/mint.json

```
@@ -42,7 +42,8 @@
         "examples/save_load_models.html",
         "examples/getting_started_complete.html",
         "examples/neuralforecast_map.html",
-        "examples/how_to_add_models.html"
+        "examples/how_to_add_models.html",
+        "examples/MLFlow_and_neuralforecast.html"
```

the names get converted to lowercase here
```suggestion
        "examples/mlflow_and_neuralforecast.html"
```